scripts/tools/zap/update_cluster_revisions.py

```diff
@@ -40,6 +44,10 @@ def getTargets():
         for filepath in Path(root).rglob('*.zap'):
             targets.append(filepath)
 
+    if cluster_id in EXCLUDE_FROM_UPDATE_DICTIONARY:
+        for target_to_exclude in EXCLUDE_FROM_UPDATE_DICTIONARY[cluster_id]:
+            targets = [target for target in targets if target_to_exclude not in str(target)]
```

Comment:
```suggestion
            targets = [target for target in targets if target_to_exclude not in target.parts]
```
should have fewer false positives.